sample-apps/javascript-sample-app/random-metrics.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS'" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+const meter = require('./meter');
+create_cfg = require('./config');
+
+const cfg = create_cfg.create_config('./config.yaml');
+const TIME_ALIVE_METRIC = 'timeAlive';
+const CPU_USAGE_METRIC = 'cpuUsage';
+const THREADS_ACTIVE_METRIC = 'threadsActive';
+const HEAP_SIZE_METRIC = 'totalHeapSize';
+
+const attributes = { statusCode: '200',  metricType: 'random' };
```

Comment:
the attributes are not according to the spec.

Reply:
Thanks. Updated.